src/microphysics/eos_phoebus/eos_phoebus.cpp

```diff
@@ -48,6 +48,23 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
     }
   };
 
+  // Temporary EOS short-circuit
+  {
+    const Real gm1 = pin->GetReal(block_name, "Gamma") - 1.0;
+    const Real Cv = pin->GetReal(block_name, "Cv");
+    const bool needs_ye = false;
+
+    EOS eos_host = IdealGas(gm1, Cv);
+    EOS eos_device = eos_host.GetOnDevice();
+
+    params.Add("d.EOS", eos_device);
+    params.Add("h.EOS", eos_host);
+    params.Add("needs_ye", needs_ye);
+  }
+
+  printf("here\n");
+  exit(-1);
```

Comment:
We need to at least also support stellar collapse. Also `exit(-1)` probably not great. :P

Reply:
I was wondering why the CI failed lol

Reply:
Fixed (I claim)